analyzer/tests/functional/analyze/test_analyze.py

```diff
@@ -98,23 +100,28 @@ def __analyze_incremental(self, content_, build_json, reports_dir,
                 os.remove(f)
         self.assertEquals(failed_file_count, failed_count)
 
+    @unittest.skipIf(version.get("gcc") is not None,
+                     "If gcc or g++ is a symlink to clang this test should be"
```

Comment:
I think we have to put a space at the end of string line if we use multi line strings otherwise there will be no white space between string lines:
```suggestion
                     "If gcc or g++ is a symlink to clang this test should be "
```
Please fix the other places too.